content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md

```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
+::: meta-data type="New" features="Asset Manager" available="late-June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.
```

Comment:
To rephrase. We need the new feature/improvement to be in the very first sentence as this is the one we will show in the communication panel of the PIM.